recipes-core/everest/everest-framework_0.20.3.bb

```
@@ -44,3 +46,17 @@ EXTRA_OECMAKE += "\
     -DPYBIND11_PYTHONLIBS_OVERWRITE=OFF \
     -DEVEREST_INSTALL_ADMIN_PANEL=OFF \
 "
+
+# disable everestrs rust support by default
+PACKAGECONFIG ??= ""
+PACKAGECONFIG[rust] = "-DEVEREST_ENABLE_RS_SUPPORT=ON -DFIND_CXXBRIDGE_BINARY=ON,-DEVEREST_ENABLE_RS_SUPPORT=OFF -DFIND_CXXBRIDGE_BINARY=OFF,cxxbridge-cmd-native,,,"
+
+do_install:append() {
+    if ${@bb.utils.contains('PACKAGECONFIG', 'rust', 'true', 'false', d)}; then
+        bbnote "everest-framework rust support enabled"
+        mkdir -p ${D}${datadir}/everest
+        cp -R ${WORKDIR}/git/everestrs ${D}${datadir}/everest/
```

Preserve, so `-rp` or the well-known `-a`.
```suggestion
        cp -a ${WORKDIR}/git/everestrs ${D}${datadir}/everest/
```